src/sentry/api/endpoints/debug_files.py

```diff
@@ -370,6 +373,29 @@ def get(self, request: Request, project) -> Response:
         return Response({"missing": missing})
 
 
+@region_silo_endpoint
+class DebugFilesConfigEndpoint(ProjectEndpoint):
+    owner = ApiOwner.HYBRID_CLOUD
+    publish_status = {"GET": ApiPublishStatus.PRIVATE}
+    permission_classes = (ProjectReleasePermission,)
+
+    def get(self, request: Request, project) -> Response:
+        org = request.organization
+        dsym_file_url = reverse(
+            "sentry-api-0-dsym-files",
+            kwargs={"organization_slug": org.slug, "project_slug": project.slug},
+        )
+
+        return Response(
+            {
+                "region_url": absolute_uri(
```

Comment:
```suggestion
                "regionUrl": absolute_uri(
```
most API attributes are camelBacked.